lib/experimental/html/class-wp-html-walker.php

```diff
@@ -231,6 +237,7 @@ public function __construct( $html ) {
 	 */
 	public function next_tag( $query = null ) {
 		$this->assert_not_closed();
+		$this->skip_rcdata();
```

Comment:
I feel like we should only skip RCDATA if we're in it. also I don't think we need to store the RCDATA state inside the walker class. if, after parsing a title or textarea tag, we skip RCDATA then, we can call `$this->skip_rcdata( $this->tag_name )` and have it jump to the end, right after twiddling our thumbs at attribute parsing.

thoughts?

Reply:
It would be minimally less optimal as you’d always have to process the rcdata, even if the textarea was the only tag you want to target

Reply:
To be fair, That doesn’t seem like a big deal

Reply:
Here's an update: https://github.com/WordPress/gutenberg/pull/42485/commits/e74d87c5360661d40e09af75c35f3eed316deeaa